CredentialsManager/Models/MenuItem.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CredentialsManager.Models
+{
+  public class MenuItem
+  {
+    public int ItemId { get; set; }
+    public string Label { get; set; }
+    public ConsoleKey ConsoleKey { get; set; }
+    public Action Action { get; set; }
```

Comment:
Since they won't ever change upon initialization, consider passing all through ctor.

Reply:
I implement the constructor in commit
[740594c](https://github.com/csinn/CSharp-From-Zero-To-Hero-v2/pull/43/commits/740594c30744a2df6e43ca5cc20e72d847cc0ef9)

Reply:
For C#9 you can do `{ get; init; }`.  Before C#9 you should at least do `{ get; private set; }`.  Or use `record`